templates/instantsearch.php

```diff
@@ -33,18 +33,18 @@
 		<article itemtype="http://schema.org/Article">
 			<# if ( data.images.thumbnail ) { #>
 			<div class="ais-hits--thumbnail">
-				<a href="{{ data.permalink }}" title="{{ data.post_title }}">
+				<a href="{{ data.permalink }}" title="{{ data.post_title }}" class="ais-hits--thumbnail-link">
 					<img src="{{ data.images.thumbnail.url }}" alt="{{ data.post_title }}" title="{{ data.post_title }}" itemprop="image" />
 				</a>
 			</div>
 			<# } #>
 
 			<div class="ais-hits--content">
-				<h2 itemprop="name headline"><a href="{{ data.permalink }}" title="{{ data.post_title }}" itemprop="url">{{{ data._highlightResult.post_title.value }}}</a></h2>
+				<h2 itemprop="name headline"><a href="{{ data.permalink }}" title="{{ data.post_title }}" class="ais-hits--title-link" itemprop="url">{{{ data._highlightResult.post_title.value }}}</a></h2>
 				<div class="excerpt">
 					<p>
 			<# if ( data._snippetResult['content'] ) { #>
-			  <span class="suggestion-post-content">{{{ data._snippetResult['content'].value }}}</span>
+			  <span class="suggestion-post-content" class="ais-hits--content-snippet">{{{ data._snippetResult['content'].value }}}</span>
```

Comment:
I assume this should be  
`class="suggestion-post-content ais-hits--content-snippet"`

and not  
`class="suggestion-post-content" class="ais-hits--content-snippet"`

Reply:
Yes you are correct.